learn/user-guide/getting-started/Installation-options.md

```diff
@@ -83,7 +85,7 @@ For instructions, see [Building from Source](/learn/user-guide/getting-started/b
 
 ## Uninstalling Ballerina
 
-Usually, the installation location is `/Library/Ballerina/distributions` in macOS, `/usr/lib64/Ballerina/distributions` in Linux, and `C:\Program Files\Ballerina\distributions` in Windows. You can either remove a particular jBallerina version or all the jBallerina versions installed in these locations.
+Usually, the installation location is `/Library/Ballerina/distributions` in macOS, `/usr/lib/ballerina/distributions` for DEB and `/usr/lib64/ballerina/distributions` for RPM in Linux, and `C:\Program Files\Ballerina\distributions` in Windows. You can either remove a particular jBallerina version or all the jBallerina versions installed in these locations.
```

Comment:
We don't need to specify up to `distributions` location. Each distribution should be removed with `bal dist remove` command.